src/services/userV2.service.ts

```diff
@@ -26,4 +26,13 @@ export class UserV2Service {
   public async getAllProducts() {
     return await this.userRepository.getProducts();
   }
+  public async getProductById(id: number) {
```

Comment:
Not a go here mate.

You can put products on a USers file.
Just make a new file
products.service.ts
and work everything there.